### ISSUE TITLE
PAL with color bleed; unclear

### ISSUE THREAD

**Issue by kattan1969** [SukkoPera/OpenVideoModulator#7]
I tested the board with both C6 and C7 capacitors soldered and ended up with a jumpy and flakey image. So, as you have suggested I decided to "play with them". Removing C7 first resulted a very bad and unstable video signal. Putting C7 back and removing C6 resulted a kind-of stable image but with terrible color bleed. Now I'm sure the chips/components I'm using are good. Could it be the video chip a Chinese one (though I purchased from the States!!). Oh and this is with PAL setup using the Composite Connector (I don't have an S-Video TV).

![image2](https://github.com/SukkoPera/OpenVideoModulator/assets/17691248/632a6e99-24fe-4b45-9fd3-f12597b8213d)
![image3](https://github.com/SukkoPera/OpenVideoModulator/assets/17691248/b64e0d39-aff0-467e-bf2a-c8a383fa4b5a)
![image1](https://github.com/SukkoPera/OpenVideoModulator/assets/17691248/eaf032c4-81cc-4956-a723-65d4f3864596)

**Comment by SukkoPera** [comment]
I'm assuming you are using the correct PAL frequency crystal and that the jumper is set to PAL.

I'd just use 100uF for C6 and 100nF for C7.

**Comment by kattan1969** [comment]
Yes for sure :) I'm using the right components.
As I've mentioned in my original post, removing C6 (the 100uF cap) resulted in the snapshots uploaded previously. Using both caps or C7 alone resulted in a very unstable and wavy composite video signal. Something is not right here!

I have a similar, pre-assembled, S-Video / Composite Video out board and the output is superb and crystal clear. It uses the same chip with many similar components as in this project (see https://www.ebay.com/itm/323934490842). 
However, this one is relatively expensive!

I was hoping to achieve the same result with this OpenVideoModulator, wasn't able so far :(

**Comment by SukkoPera** [comment]
Well, I don't know what to say, it works very well for me. Check the soldering.

Maybe you can try to analyze the other circuit and see what's different? It looks very similar at first glance.

**Comment by kattan1969** [comment]
OK, I ordered a new set of AD724JRZ chips from reputable seller based in the States. My previous set of chips (AD724JR) look different and there is a possibility that they are not genuine. Once the new chips arrive I will assemble a new PCB (as I have 10 of these) and report back the results.
I really need this circuit as it's more convenient to plug My Amiga 500 into any TV. Yet, with a very nostalgic display (yes, I know HDMI and VGA look better but having composite video out is part of the fun). I figured a way to install this inside my Amiga and use the existing mono video port for output. No case cutting other then soldering and mounting the board.

![new Chips](https://github.com/SukkoPera/OpenVideoModulator/assets/17691248/694b2d16-9468-4efa-98ef-eac22fc7c0cb)

**Comment by SukkoPera** [comment]
I have different batches of Chinese chips, some work very well, some don't work at all, some will start working properly when I touch (!) the crystal. But hey, you can get 50 for the price of a single original one ;).

**Comment by kattan1969** [comment]
It seems, at least for me, that this project is a bummer! Based on my previous comments (above), I presumed that may be the chips I purchased were fake? So I ordered new set of chips from a reputable seller and assembled a new circuit using a new PCB to result in another bad composite image that is worst than before! So whos the culprit? The chip or the circuit? I don't know!

I think this circuit is probably suitable for certain TVs with special PAL versions. I have tested this on the TVs I have at home (3 of them) and the picture is more or less the same!!! 

![thumbnail_IMG_2670](https://github.com/SukkoPera/OpenVideoModulator/assets/17691248/876efaef-1fe8-4385-84a6-31aee591a2ed)
![thumbnail_IMG_2671](https://github.com/SukkoPera/OpenVideoModulator/assets/17691248/b0adbc51-f643-45a4-a817-4e7566ca6dd3)

**Comment by SukkoPera** [comment]
The problem might be in the crystal then. Try touching it :).

(Not joking.)

**Comment by danko79** [comment]
I have the same problem, when touch the crystal with finger the interference disappears, how do solve this problem?

**Comment by SukkoPera** [comment]
No idea TBH. I thought it was a bad batch of AD724 chips but I'm not convinced. Maybe they need something different in the quartz circuit, or a different quartz, but I never found out.

If someone has any idea, please let me know! :)

**Comment by danko79** [comment]
In your opinion, the 3 220uF caps 10v too few voltage?

**Comment by SukkoPera** [comment]
No, they are OK and in any case they have nothing to do with the quartz.

**Comment by danko79** [comment]
In my case solved it by grounding the crystal body to GND (like crystal C64) now the interference is gone, definitive solution?

**Comment by SukkoPera** [comment]
That's a good suggestion, and something trivial I never tried! Let me know if it keeps working fine!

**Comment by danko79** [comment]
Sure! currently every time power on, no issue interference, isn't there another type of crystal?
